src/Codeception/Coverage/SuiteSubscriber.php

```diff
@@ -62,6 +62,7 @@ protected function applySettings($settings)
             }
         }
         $this->coverage->setProcessUncoveredFilesFromWhitelist($this->settings['show_uncovered']);
+        $this->coverage->setAddUncoveredFilesFromWhitelist(false);
```

Comment:
It looks like this line doesn't matter or it isn't covered by test.
The change in Subscriber/Printer class is enough to make your tests pass.